apps/mobile/src/components/Badge/theme.ts

```diff
@@ -33,14 +33,6 @@ export const badgeTheme = {
     color: tokens.color.warning1ContrastTextDark,
     background: tokens.color.warningBackgroundDark,
   },
-  light_badge_warning_variant1: {
-    color: tokens.color.warning1TextLight,
-    background: tokens.color.warning1ContrastTextLight,
-  },
-  dark_badge_warning_variant1: {
-    color: tokens.color.warning1ContrastTextDark,
-    background: tokens.color.warningDarkDark,
-  },
```

Comment:
so there is no place in the codebase where we would use warning_variant1 anymore?